src/Nethermind/Nethermind.JsonRpc.Test/Modules/SubscribeModuleTests.cs

```diff
@@ -1055,7 +1050,7 @@ public void SyncingSubscription_on_NewHeadBlock_event_when_sync_changed_to_true(
 
             jsonRpcResult.Response.Should().NotBeNull();
             string serialized = _jsonSerializer.Serialize(jsonRpcResult.Response);
-            var expectedResult = string.Concat("{\"jsonrpc\":\"2.0\",\"method\":\"eth_subscription\",\"params\":{\"subscription\":\"", syncingSubscription.Id, "\",\"result\":{\"isSyncing\":true,\"startingBlock\":\"0x0\",\"currentBlock\":\"0x2728\",\"highestBlock\":\"0x273a\"}}}");
```

Comment:
Accidental over sharing of internal value; if its syncing it returns the data; otherwise `false`; not api docs on infrua, alchemy or quicknode have this property